assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/nlb.tf

```diff
@@ -36,30 +36,6 @@ resource "aws_lb_listener" "apiserver-https" {
   }
 }
 
-# Forward HTTP ingress traffic to workers
```

Comment:
There is a typo in commit message: `protocl`.

Reply:
Thanks, fixed.